### PR TITLE
REF: Multisig Help to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/akjv5a43msz2sialw2eew43y4a?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12243)

File: screen/wallets/addMultisigHelp.tsx [hunk 1]
```diff
@@ -32,43 +30,34 @@ const WalletsAddMultisigHelp = () => {
...
+  return (
     <SafeBlueArea style={stylesHook.root}>
       <ScrollView>
         <View style={[styles.intro, stylesHook.intro]}>
           <Text style={[styles.introTitle, stylesHook.introTitle]}>{loc.multisig.ms_help_title}</Text>
           <Text style={[styles.introText, stylesHook.introText]}>{loc.multisig.ms_help_text}</Text>
           <Image style={styles.introImage} source={require('../../img/mshelp/mshelp-intro.png')} />
         </View>
-        <View style={[styles.tip, stylesHook.tip]}>
-          <Text style={[styles.introTip, stylesHook.introTip]} />
```
Comment (Overtorment) [inline screen/wallets/addMultisigHelp.tsx:50]: did this <Text> have a purpose? maybe, create space or smth?

**Review by Overtorment — APPROVED** [review]

File: screen/wallets/addMultisigHelp.tsx (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/addMultisigHelp.tsx:50]: No idea i left that code alone

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**